plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeAnalyze.java

```diff
@@ -147,22 +139,24 @@ private void testAnalyze(Optional<Integer> checkpointInterval)
     public void testAnalyzePartitioned()
     {
         String tableName = "test_analyze_" + randomNameSuffix();
-        assertUpdate("CREATE TABLE " + tableName
+        assertUpdate(
```

Comment:
Please do create a compatibility test with spark to verify that after a CTAS `DESC EXTENDED` works as intended on Databricks

Reply:
Nevermind. Trino Delta Lake (on the storage layer) & Databricks (on the metastore properties) have outputs in different places.